### ISSUE TITLE
Image signature generated by PHP client is incorret

### ISSUE THREAD

**Issue by wujekbogdan** [imagekit-developer/imagekit-php#54]
Let's compare the results of node SDK and PHP SDK calls.

``` js
const imageURL = imagekit.url({
  path: '/default-image.jpg',
  signed: true,
  transformation: [{
    'height': 300,
    'width': 400
  }],
});
```
gives:

```
https://ik.imagekit.io/${my_identifier}/tr:h-300,w-400/default-image.jpg?ik-s=29824aebb89cc07e5fb0561e748e2bccb5f83855
```

``` php
$url = $client->getUrl([
    'path' => '/default-image.jpg',
    'signed' => true,
    'transformation' => [
        [
            'width' => 300,
            'height' => 400,
        ],
    ],
]);
```

gives:

```
https://ik.imagekit.io/${my_identifier}/tr:w-300,h-400/default-image.jpg&ik-s=e4adef31d412d9635efb446bf19792dfea1609b
```

As you can see the signs are different which results in `401 Unauthorized: Invalid request signature` response when requesting the URL generated by the PHP client.

In both cases, I use the same credentials.

**Comment by wujekbogdan** [comment]
I think I know what's going on. [This bug seems to be fixed in `v3.0.1`](https://github.com/imagekit-developer/imagekit-php/issues/48), but this version doesn't appear to be released to [packagist.org](https://packagist.org/packages/imagekit/imagekit) so the `"imagekit/imagekit": "^3.0"` entry in `composer.json` results in version `3.0.0.` rather than `3.0.1`.

// EDIT:
Confirmed: Forcing composer to install the `dev-master#3.0.1` version solves the problem:

``` json
{
    "imagekit/imagekit": "dev-master#3.0.1"
}
```

 I will keep this ticket open though because the issue applies to the most recent publicly available version.

**Comment by imagekitio** [comment]
Please use 3.0.2 - https://packagist.org/packages/imagekit/imagekit#3.0.2
